tests/sanity/tsconfig.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
Why it is removed?

Reply:
OK, but this did not affect the operation of autotests. In general, it is enough to specify other directories (to which I moved the root directory and out dir).